javascript/MaterialXView/index.ejs

```diff
@@ -37,6 +37,5 @@
       <canvas id="webglcanvas" tabindex="1" style="outline: none;"></canvas>
     </div>
     <script src="JsMaterialXGenShader.js"></script>
-    <script src="main.js"></script>
```

Comment:
This should never have been there as it invokes 2 instances to run causing all evaluation and load to occur twice.

Reply:
Good catch, @kwokcb!